server/lambda/layer/common/nodejs/custom_modules/post-to-slack.js

```diff
@@ -17,33 +17,43 @@ const generateStreamLink = () => {
 const formatTime = (start) => {
   if (!start) return 'N/A';
   const elapsed = (new Date().getTime() - start) / 1000;
-  return `${Math.floor(elapsed / 60)}m${Math.round(elapsed % 60)}s`;
+  return `${Math.floor(elapsed / 60)}m${(elapsed % 60).toFixed(3)}s`;
+};
+
+const formatFunctionName = () => {
+  if (!process.env.AWS_LAMBDA_FUNCTION_NAME || !process.env.AWS_LAMBDA_FUNCTION_VERSION) return 'N/A';
+  return `${process.env.AWS_LAMBDA_FUNCTION_NAME}:${process.env.AWS_LAMBDA_FUNCTION_VERSION}`;
 };
 
 const formatMessage = (title, time, message) => {
-  const link = generateStreamLink();
-  return `{ "blocks": [
-    {
-      "type": "section",
-      "text": {
-        "type": "mrkdwn",
-        "text": "Lambda Job: *${title}*\nFunction Name: ${process.env.AWS_LAMBDA_FUNCTION_NAME}:${process.env.AWS_LAMBDA_FUNCTION_VERSION}\nElapsed Time: ${time}\n\nFull output included below."
+  const escapedMessge = typeof message === 'string' ? message : JSON.stringify(message);
+  const payload = {
+    blocks: [{
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text: `Lambda Job: *${title}*\nLambda Function Name: ${formatFunctionName()}\nElapsed Time: ${time}\n\nFull output included below.`,
```

Comment:
The PHAC Submission slack notification is coming with a Lambda Job label despite not being one, probably a more general name here?

Reply:
Good call. Fixed